CONTRIBUTORS.md

```diff
@@ -128,3 +128,4 @@ Cerner Corporation
 [@alex-bezek]: https://github.com/alex-bezek
 [@BenBoersma]: https://github.com/BenBoersma
 [@CoryMcDonald]: https://github.com/CoryMcDonald
+[@DineshSingh9]: https://github.com/DineshSingh9
```

Comment:
Your name should be added to the section above username like `- <Name> [@<Username>]`